.github/workflows/auto-copilot-review.yml

```
@@ -0,0 +1,70 @@
+      pull-requests: write
+      contents: read
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.workflow_run.head_sha }}
+
+      - name: Get PR number from head SHA
+        id: get-pr
+        uses: actions/github-script@v7
```

actionlint (1.7.11)

[error] missing input "script" which is required by action "actions/github-script@v7". all required inputs are "script"

(action)